lib/src/base-styles/_nx-text-input.scss

```diff
@@ -6,32 +6,52 @@
  */
 
 .nx-text-input {
-  border: $nx-border-dark;
-  border-left-width: 4px;
-  border-radius: 0;
-  box-shadow: none;
-  height: $nx-form-element-height;
-  padding: 0 8px;
   width: $nx-form-element-width-normal;
+}
+
+.nx-text-input__box {
+  align-items: center;
+  border: 1px solid #d1d6f0;
```

Comment:
I believe this border color is only supposed to apply in the pristine state

Reply:
> I believe this border color is only supposed to apply in the pristine state

In Zeplin it shows that color for the default state.